test/test_mailer_server.js

```diff
@@ -2,54 +2,23 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-/*eslint no-console: 0*/
-
-var P = require('../lib/promise')
-var request = require('request')
 var createDBServer = require('fxa-auth-db-mysql')
 
 function TestServer(config) {
   this.config = config
   this.server = null
 }
 
-function waitLoop(testServer, url, cb) {
```

Comment:
It is unrelated, so it is in another commit. It was something I noticed I had missed from the earlier remote test refactoring. Just felt :shrug: about filing a separate PR.